src/client/datascience/jupyter/liveshare/hostJupyterServer.ts

```diff
@@ -146,35 +138,30 @@ export class HostJupyterServer implements INotebookServer {
         return launchInfo;
     }
 
-    public async connect(launchInfo: INotebookServerLaunchInfo, _cancelToken?: CancellationToken): Promise<void> {
-        traceInfo(`Connecting server ${this.id}`);
+    public async connect(connection: IJupyterConnection, _cancelToken?: CancellationToken): Promise<void> {
```

Comment:
Dumped INotebookServerLaunchInfo in favor of IJupyterConnection (none of the properties of INotebookServerLaunchInfo as used execept for connectionInfo , mentioend above)